templates/documents/partials/download_progress.html

```diff
@@ -0,0 +1,111 @@
+{% load static %}
+
+<div class="w-full">
+  <div id="progress-container" class="flex flex-col gap-3 p-4 bg-base-200 rounded-lg">
+    <div id="progress-header" class="flex items-center gap-2">
+      <span class="loading loading-spinner loading-sm"></span>
+      <h3 class="font-semibold">Creating ZIP file...</h3>
+    </div>
+
+    <div class="progress-wrapper progress w-full bg-base-300 rounded-full overflow-hidden h-4">
+      <div id="progress-bar" class="progress-bar h-full bg-primary transition-all duration-300 ease-out"></div>
+    </div>
+    <div id="progress-bar-message" class="text-sm text-center text-base-content/70">
+      Waiting for progress to start...
+    </div>
+  </div>
+
+  <div id="download-complete-section" class="hidden mt-4">
+    <div class="alert alert-success">
+      <i class="fa-solid fa-check-circle"></i>
+      <div class="flex-1">
+        <h3 class="font-bold">ZIP file ready!</h3>
+        <div class="text-sm">Your download is ready. This file will be available for 24 hours.</div>
```

Comment:
nit: remove the 24 availability text since there's no way for the user to access this once they click the 'done' button.
```suggestion
        <div class="text-sm">Your download is ready.</div>
```